src/screens/cryptocurrency/SelectCryptocurrency.tsx

```diff
@@ -0,0 +1,57 @@
+import { FlashList } from "@shopify/flash-list";
+import { ActivityIndicator, StyleSheet, View } from "react-native";
+
+import Container from "@/components/Container";
+import TokenData from "@/components/listItems/TokenData";
+import { accent, spacing } from "@/constants/DesignTokens";
+import useGetTokens from "@/hooks/useGetTokens";
+
+type Event = { id: string; symbol: string };
+
+type SelectCryptocurrencyProps = {
+  OnSelectCryptocurrency: (event: Event) => void;
+};
+
+export default function SelectCryptocurrency({
```

Comment:
It should be capitalized

```suggestion
export default function SelectCryptoCurrency({
```